### PR TITLE
how to read the report

### PR DESCRIPTION
**Pull request by maelle**
cf #33

### CODE REVIEW

File: vignettes/how-to-read-covrpage-report.Rmd [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Now, in the `covrpage` report.
+
+* Look at distribution of coverage (table 1)
+    * check if there are whole groups of functionalities that were not tested.
+    * if you need the line by line coverage go to Codecov/Coveralls (linked from the test coverage badge) and dig in there. 
+
+* look at test stats in (table 2)
+    * summary of what tests contain a lot of skipped/warning, is it concentrated in only one or is it all over the tested files.
+
+* look at breakdown of tests (table 3)
```
Comment (yonicd) [inline vignettes/how-to-read-covrpage-report.Rmd]: this table is hidden on a successful build and is open if any test is non-passing. it is a simple visual cue that informs the reader if something is wrong. Beyond this there are the icons that focus the eyes to where in the tests tables to look.

File: vignettes/how-to-read-covrpage-report.Rmd [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+`covrpage` can help you choose whether to trust a package. As a potential user of a package whose repo or website features a `covrpage` report, this vignette will help you understand how can you use that information to make an informed decision on:
+
+* whether to use the package at all, or right now;
+
+* whether to stay away from some of its functionalities for the time being?
+
+As a package developer, this same information can help you assess what trustworthiness your own package is radiating.
+
+# Case 1: package with a passing Travis build
```
Comment (yonicd) [inline vignettes/how-to-read-covrpage-report.Rmd]: doesn't have to be ci (travis) build, could be built locally.

Comment (maelle) [reply vignettes/how-to-read-covrpage-report.Rmd]: ok but I was imagining someone external looking at the report (i.e. without needing to clone and run anything locally). will edit.

**Comment by maelle** [comment]
I am not sure why CI fails.

File: vignettes/how-to-read-covrpage-report.Rmd [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    * see if any throw warnings or are skipped, and check why (skip_travis/skip_cran)
+    * use links to jump to actual problem tests/expectations to investigate further
+    
+# Case 2: package with a failing Travis build/local R CMD check
+
+(i.e. a red Travis badge)
+
+In the `covrpage` report,
+
+* you'll see a disclaimer at the top if tests failed, see [this example](https://github.com/llrs/GSEAdv/tree/19cbab3cff04c5c65bcef18c493a2cc650f441c2/tests#coverage).
```
Comment (yonicd) [inline vignettes/how-to-read-covrpage-report.Rmd]: do we want to show a "real" or a "dummy" repo (ie covrfail)?

Comment (maelle) [reply vignettes/how-to-read-covrpage-report.Rmd]: your call :-)

Comment (yonicd) [reply vignettes/how-to-read-covrpage-report.Rmd]: i'll leave it as it...

**Comment by maelle** [comment]
Any further comments?

**Pull request merged**